sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -598,6 +598,16 @@ object SQLConf {
       .bytesConf(ByteUnit.BYTE)
       .createOptional
 
+  val ADAPTIVE_SHUFFLE_HASH_JOIN_LOCAL_MAP_THRESHOLD =
+    buildConf("spark.sql.adaptive.shuffledHashJoinLocalMapThreshold")
+      .doc("Configures the maximum size in bytes for per partition that can be allowed to build " +
+        "local hash map. If all the partition size not larger than this threshold, join " +
+        "selection may use shuffled hash join instead of sort merge join when " +
+        s"${PREFER_SORTMERGEJOIN.key} is false.")
```

Comment:
really? Shall we use shuffle hash join regardless of this `PREFER_SORTMERGEJOIN` config?

Reply:
yeah, right. I missed update this docs.